optaplanner-benchmark/src/build/revapi-config.json

```diff
@@ -0,0 +1,95 @@
+{
+  "revapi": {
+    "java": {
+      "filter": {
+        "packages": {
+          "comment": "We don't want to check transitive classes, since we already check them in their own module.",
+          "regex": true,
+          "include": [
+            "org\\.optaplanner\\.benchmark\\.api.*",
+            "org\\.optaplanner\\.benchmark\\.config.*"
```

Comment:
For config classes, I think the regex should only look at the fields (names and types) (optionally including the getters and setters), but not at the build and other methods, as those aren't guaranteed to be backwards compatible.